readme.md

```diff
@@ -39,11 +80,39 @@ Run:
 ## Documentation
 Refer to [documentation](docs/lioncore2mps-converter-design.adoc).
 
-## Publishing
+## Development process
+
+We have on "main" branch per supported MPS version, e.g. `mps2021.1`.
+We implement all new functionality on a branch, based on the oldest supported "main" branch.
```

Comment:
Maybe mention that within these major.minor version you generally should use the latest patch version?